apps/tailwind-components/components/Input.vue

```diff
@@ -0,0 +1,116 @@
+<script setup lang="ts">
+import type { IInputProps, IValueLabel } from "~/types/types";
+import type { CellValueType } from "metadata-utils/src/types";
+const modelValue = defineModel<any>();
+const props = withDefaults(
+  defineProps<
+    IInputProps & {
+      type: CellValueType;
+      label?: string;
+      description?: string | null;
+      describedBy?: string;
+      required?: boolean;
+      errorMessage?: string | null;
+      refSchemaId?: string;
+      refTableId?: string;
+      refLabel?: string;
+      options?: IValueLabel[];
+    }
+  >(),
+  {
+    required: false,
+  }
+);
+const emit = defineEmits(["focus", "blur", "update:modelValue"]);
+const TYPE = computed(() => props.type.toUpperCase());
+</script>
+
+<template>
+  <LazyInputString
+    v-if="['STRING', 'AUTO_ID', 'LONG', 'EMAIL', 'INT'].includes(TYPE)"
```

Comment:
Email and password should be split out so they can have 'type="email"' or 'type="password"'

Reply:
or extra prop for string inputs